pol-is/polisClientAdmin
# Homepage v2 lite

## Vision

🚨 Review this PR carefully! Lots of Changes 🚨 

This PR revamps homepage and various lander pages. It's the first of many towards a v2 refresh of Pol.is website. 

### Notable Changes

- Removing a fair number of lander pages that were not used. I opted to leave some (eg. gov, bot) because I'm not sure if they have been referenced
- Introduced Tachyons library for styling. Slowly pulling out Radium
- New components follow React best practices of naming folders and components with Capital letters

## Todo

- [x] Add icons to homepage
- [ ] Test new request demo page
- [x] Add hero image
- [x] Tighten up a few styles (if time)
- [x] Cleanup/Rebase a few commits for better log
- [ ] React Helmet (Bonus, if time. It wasn't in before so I consider this an improvement and not a missing piece of functionality)

## To Test

1. run `npm install`
1. start the app
1. click around and make sure links work
1. visit `/demo` and fill in the form. ensure it still submits correctly
